po/pl.po

```diff
@@ -517,13 +519,13 @@ msgid "Exporting list of read articles..."
 msgstr "Eksportowanie listy przeczytanych artykułów..."
 
 #: src/controller.cpp:572
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "%<PRIu64> unreachable feeds found. See `cleanup-on-quit` in newsboat(1) for "
 "details."
 msgstr ""
-"Znaleziono nieosiągalne kanały, rozważ ustawienie `cleanup-on-quit yes` lub "
-"uruchom `newsboat --cleanup`"
+"%<PRIu64> nieosiągalnych kanałów. Więcej informacji pod  `cleanup-on-quit` w "
```

Comment:
I know nothing about Polish. There are two spaces in between `pod` and ````cleanup-on-quit````. Is this intentional or should it be just one?

Reply:
Nice catch. No, it was not intentional, should be fixed now.